server/lib/poller.js

```diff
@@ -65,12 +65,12 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
+            checkAccounts = await Config.findOrCreateDefaultBooleanValue(
                 'weboob-auto-merge-accounts'
             );
-
+        } finally {
```

Comment:
I think you can keep the previous code structure, without a `finally` statement here. Having `try/catch` inside a `try/catch` should work.